pilot/pkg/xds/endpoints/mtls_checker.go

```diff
@@ -53,6 +53,8 @@ func (c *mtlsChecker) checkMtlsEnabled(ep *model.IstioEndpoint) bool {
 		switch *drMode {
 		case networkingapi.ClientTLSSettings_DISABLE:
 			return false
+		case networkingapi.ClientTLSSettings_SIMPLE:
```

Comment:
I think anything but mutual is false (per https://github.com/istio/istio/issues/46555#issuecomment-1692256637)

Reply:
So no MUTUAL, only ISTIO_MUTUAL? Check that it's correct once more before I merge.

Reply:
yep